snippets/python/src/closed_channel.py

```diff
@@ -0,0 +1,20 @@
+import breez_sdk
+def prepare_sweep(fee_rate):
+    # ANCHOR: prepare_sweep
+    destination_address = "bc1.."
+    sat_per_vbyte = fee_rate
+    try:
+        req = breez_sdk.PrepareSweepRequest(fee_rate,  destination_address)
```

Comment:
Have to be careful of the ordering in python, it takes the to_address as the first param
```suggestion
        req = breez_sdk.PrepareSweepRequest(destination_address, fee_rate)
```

Reply:
Ordered